### PR TITLE
Add test for QueryManager continuing to poll after refetch (see issue #239)

### PR DESCRIPTION
**Pull request by ciwolsey**
Test regarding issue #239. Hope this is okay.

### CODE REVIEW

**Comment by stubailo** [comment]
Thanks very much!

**Comment by abhiaiyer91** [comment]
Im down to solve this one if @ciwolsey doesn't want to

**Comment by ciwolsey** [comment]
Added a fix that greens the test too but I'm definitely not sure I've done it in the best way.

File: src/QueryManager.ts [hunk 1]
```diff
@@ -220,13 +220,6 @@ export class QueryManager {
           this.stopQuery(queryId);
         },
         refetch: (variables: any): Promise<GraphQLResult> => {
-          // if we are refetching, we clear out the polling interval
```
Comment (abhiaiyer91) [inline src/QueryManager.ts]: why was this removed?

Comment (stubailo) [inline src/QueryManager.ts]: The problem we're trying to fix here is that calling `refetch` currently kills polling completely, but that's not the intended outcome - ideally the client will keep polling after the forced refetch.

**Comment by helfer** [comment]
@ciwolsey I think this does what we need, but the linter checks are not passing. Can you look into that?

**Comment by helfer** [comment]
@ciwolsey Any update on this? It seems the only thing that you'd have to fix is this:

`test/QueryManager.ts[753, 9]: if statements must be braced`

**Comment by ciwolsey** [comment]
Fixed the if statement

**Comment by helfer** [comment]
@ciwolsey Thanks a lot! Could you do a rebase as well? We've merged some other PRs into master since you made this one.

**Comment by helfer** [comment]
Merged! Thanks a lot!
